.github/workflows/release.yml

```diff
@@ -25,6 +25,16 @@ jobs:
           fetch-depth: 0
 
       - name: Prepare release
+        id: new-version
         uses: ./release
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Update major version branch
+        run: |
+          set -ex
+          v="${new_tag%%.*}"
+          git update-ref refs/heads/"$v"
+          git push origin "$v"
+        env:
+          new_tag: ${{ steps.new-version.outputs.new-tag }}
```

Comment:
Where/how  new-tag value comes to be known?

Reply:
It's output by the `datalad_release_action release` command [here](https://github.com/datalad/release-action/blob/47fee76ba156920ebd8146f518f73929c2b061d4/datalad_release_action/__main__.py#L131) and then declared as an action output [here](https://github.com/datalad/release-action/blob/47fee76ba156920ebd8146f518f73929c2b061d4/release/action.yml#L20).